examples/fabric-bridge-app/fabric-bridge-common/src/ZCLCallbacks.cpp

```diff
@@ -34,10 +34,9 @@ Protocols::InteractionModel::Status emberAfExternalAttributeReadCallback(Endpoin
                                                                          const EmberAfAttributeMetadata * attributeMetadata,
                                                                          uint8_t * buffer, uint16_t maxReadLength)
 {
-    uint16_t endpointIndex  = emberAfGetDynamicIndexFromEndpoint(endpoint);
     AttributeId attributeId = attributeMetadata->attributeId;
 
-    Device * dev = DeviceMgr().GetDevice(endpointIndex);
+    Device * dev = DeviceMgr().GetDevice(endpoint);
```

Comment:
Totally unrelated to this PR, but the SDK already has a device manager class, and this one doesn't look to be namespaced. You might want to namespace this one or re-name it (BridgedDeviceManager?)

Reply:
I will address this in a follow up

Reply:
https://github.com/project-chip/connectedhomeip/pull/34776